common/openapi.yml

```diff
@@ -23,126 +23,167 @@ paths:
       requestBody:
         required: true
         content:
           application/json:
             schema:
-              $ref: '#/components/schemas/PostJobRequestBody'
+              $ref: '#/components/schemas/CreateJobRequestBody'
       responses:
-        '200':
-          description: Response to createJob calls
+        200:
+          description: Success
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/PostJobResponse'
-        '400':
+                $ref: '#/components/schemas/CreateJobResult'
+        400:
           $ref: '#/components/responses/BadRequest'
-        '403':
+        403:
           $ref: '#/components/responses/PermissionDenied'
-        '404':
-          $ref: '#/components/responses/NotFound'
-        '500':
+        404:
           $ref: '#/components/responses/NotFound'
+        500:
+          $ref: '#/components/responses/ServerError'
 
-  /api/jobs/v1alpha1/{pipelineId}/{jobId}:
+  /api/pipelines/v1alpha1/{pipelineId}/jobs:
     parameters:
       - $ref: '#/components/parameters/PipelineId'
-      - $ref: '#/components/parameters/JobId'
+      - $ref: '#/components/parameters/Limit'
+      - $ref: '#/components/parameters/PageToken'
     get:
-      summary: Retrieve specified job
-      operationId: getJob
-      tags: [ jobs ]
+      summary: Retrieve all pipeline-specific jobs requested by user
+      operationId: getPipelineJobs
+      tags: [ pipelines ]
       responses:
-        '200':
+        200:
           description: Success
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/GetJobResponse'
-        '403':
+                $ref: '#/components/schemas/GetJobsResponse'
+        403:
           $ref: '#/components/responses/PermissionDenied'
-        '404':
+        404:
           $ref: '#/components/responses/NotFound'
-        '500':
+        500:
           $ref: '#/components/responses/ServerError'
 
-  /api/pipelines/v1alpha1:
+  /api/job/v1alpha1/jobs:
     get:
-      summary: Return all available Pipelines
-      operationId: getPipelines
-      tags: [ pipelines ]
+      tags: [ jobs ]
+      description: >-
+        Returns a list of all of the jobs the caller has access to
+      operationId: getAllJobs
+      parameters:
+        - $ref: '#/components/parameters/Limit'
+        - $ref: '#/components/parameters/PageToken'
       responses:
-        '200':
-          description: Success
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/PipelinesGetResult'
-        '403':
+        200:
+            description: Success
+            content:
+              application/json:
+                schema:
+                  $ref: '#/components/schemas/GetJobsResponse'
+        403:
           $ref: '#/components/responses/PermissionDenied'
-        '404':
+        404:
```

Comment:
shouldnt need 403 or 404 here if we're just returning the list of jobs

Reply:
good catch, thank you!